.slashrc

```diff
@@ -92,6 +93,7 @@ class MediaPlugin(slash.plugins.PluginInterface):
     self.ctapt = args.ctapt
     self.ctapr = args.ctapr
     self.platform = args.platform
+    self.render_device = args.device
```

Comment:
Let's do an assert after this to ensure the device exists (i.e. os.path.exists).